queries/play.py

```diff
@@ -0,0 +1,372 @@
+            "settu á fóninn",
+            "settu á fóninn fyrir mig",
+            "geturðu spilað",
+            "getur þú spilað fyrir mig",
+            "gætirðu spilað",
+            "gætir þú spilað fyrir mig",
+            "viltu spila",
+            "viltu spila fyrir mig",
+            "vilt þú spila",
+            "vilt þú spila fyrir mig",
+            "nennirðu að spila",
+            "nennirðu að spila fyrir mig",
+            "nennir þú að spila",
+            "nennir þú að spila fyrir mig",
+        )
+    )
+)
+
+_ADJ = "|".join(
+    frozenset(
+        (
+            "góðan",
+            "góða",
+            "gott",
+            "góð",
+            "skemmtilegan",
+            "skemmtilegt",
+            "skemmtilega",
+            "skemmtileg",
+            "huggulegan",
+            "huggulega",
+            "huggulegt",
+            "hugguleg",
+            "einhvern",
+            "eitthvað",
+            "einhverja",
+        )
+    )
+)
+
+_POST = "|".join(frozenset(("fyrir mig", "fyrir okkur")))
+
+
+REGEX_Q2H = OrderedDict(
+    {
+        # Any music
+        r"^(?:{0})\s?(?:{1})?\s(tónlist|tónverk|lag|slagara)\s?(?:{2})?$".format(
+            _VERB, _ADJ, _POST
+        ): _play_music,
+        # Jazz
+        r"^(?:{0})\s?(?:{1})?\s(djass|jazz|jass|djasstónlist|djasslag)\s?(?:{2})?$".format(
+            _VERB, _ADJ, _POST
+        ): _play_jazz,
+        # Blues
+        r"^(?:{0})\s?(?:{1})?\s(blús|blúsinn|blústónlist|blúslag)\s?(?:{2})?$".format(
+            _VERB, _ADJ, _POST
+        ): _play_blues,
+        # Rock
+        r"^(?:{0})\s?(?:{1})?\s(rokk|rokktónlist|rokklag|rokk og ról)\s?(?:{2})?$".format(
+            _VERB, _ADJ, _POST
+        ): _play_rock,
+        # Classical
+        r"^(?:{0})\s?(?:{1})?\s(klassíska tónlist|klassík|klassískt)\s?(?:{2})?$".format(
+            _VERB, _ADJ, _POST
+        ): _play_classical,
+        # Electronic
+        r"^(?:{0})\s?(?:{1})?\s(raftónlist|elektróníska tónlist|elektrónískt)\s?(?:{2})?$".format(
+            _VERB, _ADJ, _POST
+        ): _play_electronic,
+        # Play music by X
+        r"^(?:{0})\s?(?:{1})?\s(?:tónlist|tónverk|lag|verk|slagara) (?:eftir|með|í flutningi) (.+)$".format(
+            _VERB, _ADJ
+        ): _play_music_by_artist,
+        # Play song Y by X
+        r"(?:{0}) (.+) (?:eftir|með|í flutningi) (.+)$".format(
+            _VERB
+        ): _play_song_by_artist,
+    }
+)
+
+
+def handle_plain_text(q: Query) -> bool:
+    """Handle a plain text query."""
+    ql = q.query_lower.rstrip("?")
+
+    # Check if it's a hardcoded barestring query
+    handler_fn = HARDCODED_Q2H.get(ql)
+    if handler_fn:
+        handler_fn(ql, q)
+        return True
+
+    # Check if query matches regexes supported by this module
+    matches = None
+    for rx, fn in REGEX_Q2H.items():
+        matches = re.search(rx, ql)
+        if matches:
+            fn(ql, q, matches)
+            break
+
+    if not matches:
```

Comment:
This could actually be an else: clause on the for statement. Just sayin'

Reply:
I don't like else clauses in for statements :)